.github/workflows/backport.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     steps:
       - name: GitHub App token
         id: github_app_token
-        uses: tibdex/github-app-token@v1.5.0
+        uses: tibdex/github-app-token@v1
```

Comment:
Just for my understanding, will mend take care of upgrading to major version directly?

Reply:
> will mend take care of upgrading to major version directly?

Yep, see for example #403